hub/log.go

```diff
@@ -1,44 +1,24 @@
 package hub
 
-import (
-	fluentd "github.com/joonix/log"
-	log "github.com/sirupsen/logrus"
-	"github.com/spf13/viper"
-)
+import "go.uber.org/zap/zapcore"
 
-func addUpdateFields(f log.Fields, u *Update, debug bool) log.Fields {
-	f["event_id"] = u.ID
-	f["event_type"] = u.Type
-	f["event_retry"] = u.Retry
-	f["update_topics"] = u.Topics
-	f["update_private"] = u.Private
+type stringArray []string
```

Comment:
I would probably move this to a package of its own. It's an implementation and is not really related to the hub package apart from the fact that it consumes it.

Reply:
It's purely internal (and copied from Zap actually).